tests/config/astrometryReferenceErr-None-config.py

```diff
@@ -3,4 +3,5 @@
 # We have to use the sdss refcat, as gaia supplies coord errors.
 config.astrometryRefObjLoader.ref_dataset_name = "sdss-dr9-fink-v5b"
 config.astrometryRefObjLoader.filterMap = {}
+config.astrometryRefObjLoader.anyFilterMapsToThis = None
```

Comment:
Is this necessary, since None is the default? Please consider a brief code comment.